.github/workflows/cypress.yml

```diff
@@ -1,8 +1,6 @@
 name: Run cypress at 7am every day
```

Comment:
Should this not be removed also?

Reply:
so I didn't added a description I am just removing from the package json but I will keep the jobs